doc/release/0.2.rst

```diff
@@ -48,6 +48,7 @@ Bug fixes
 #. Fixed a bug in database comparison. (`PR#1258 <https://github.com/terrapower/armi/pull/1258>`_)
 #. Fixed an invalid assumption on the lattice physics and cross section manager interfaces when using tight coupling for snapshot runs. (`PR#1206 <https://github.com/terrapower/armi/pull/1206>`_)
 #. Fixed a bug where the precision used to determine the axial submesh was too small. (`PR#1225 <https://github.com/terrapower/armi/pull/1225>`_)
+#. Fixed a bug in the ISOTXS file name used for snapshots. (`PR#1277 <https://github.com/terrapower/armi/pull/1277>`_)
```

Comment:
Damn, I was going to merge this PR.

But can you move this release note to the most recent version of ARMI (v0.2.8 above)?

Thanks! Sorry!

Reply:
I would fix it myself, but you opened this PR from your private fork repo.

Reply:
Fixed, you can review now!